amaznot/resources/views/layouts/layout.blade.php

```diff
@@ -1,81 +1,248 @@
+          <button class="btn dropdown-toggle shadow-none currency" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            CAD
+          </button>
+          <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+            <a class="dropdown-item change-dropdown" href="#">CAD</a>
+            <a class="dropdown-item change-dropdown" href="#">USD</a>
+          </div>
+        </div>
+      </div>
+      <div class="controller-container div-inline">
+        <div class="profile div-inline mr-4">
+          <a class="controller-link" href="{{ route('login') }}">
+            <i class="fas fa-user"></i>
+            Login
+          </a>
+        </div>
+        @if(Illuminate\Support\Facades\Auth::check())
+        <div class="profile div-inline">
+          <a class="controller-link" href="{{ route('cart') }}">
+            <i class="fas fa-shopping-cart"></i>
+            Items
+          </a>
+        </div>
+        <p class="controller-total-price">
+          $ 0.00
+        </p>
+        @endif
+      </div>
+    </div>
+    <div class="custom-bottom-nav">
+      <nav class="navbar navbar-expand-lg navbar-light">
+        <a class="navbar-brand" href="{{route('home')}}">
+          <img src="{{asset('assets/amaznot_full.png')}}" width="100" height="30" class="d-inline-block align-top logo-img" alt="Logo" loading="lazy">
+        </a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
 
-    <!--HEADER-->
-    <!--END HEADER-->
+        <div class="collapse navbar-collapse ml-auto navbar-items-container" id="navbarSupportedContent">
+          <ul class="navbar-nav ml-auto navbar-items">
+            <li class="nav-item">
+              <a class="nav-link" href="{{ route('home') }}">Home</a>
+            </li>
+            <li class="nav-item dropdown position-static">
+              <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                Aisles
+              </a>
+              <div class="dropdown-menu w-100" aria-labelledby="navbarDropdown">
+                <div class="nav-categories-container">
+                  <div class="category-item">
+                    <h5 class="category-item-header">
+                      Choose a Category
+                    </h5>
+                    @php
+                    $data = array_map(fn($data) => $data['category'], $headerData['array']);
+                    $data = array_unique($data);
+                    @endphp
+                    @foreach ($data as $cat)
+                    @php
+                    $catURL = "https://".$_SERVER['SERVER_NAME']."/products/".$cat;
```

Comment:
This may be an issue with an installation of mine, but this manner of formatting links fails on my localhost, as it attempts to bring me from localhost:8000 to 127.0.0.1. Would it not be possible to have these links created with route helpers, as the others are? Or does the filtering introduce a lot of complexity to that?